docs/react-testing-library/example-intro.md

```diff
@@ -122,12 +122,12 @@ export default function Fetch({ url }) {
   const buttonText = buttonClicked ? 'Ok' : 'Load Greeting'
 
   return (
-    <div>
+    <>
```

Comment:
Let's not use too much syntax here. There are more people familiar with `<div>` than `<>`. What was the purpose of this change?